src/XenobladeChroniclesX/Mods/BattleDamageModGround2/rules.txt

```diff
@@ -0,0 +1,28 @@
+[Definition]
+titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
+name = "Ground damage divisor
```

Comment:
You're missing the ending " here, although we usually just use it without quotes so I'd change it to
```suggestion
name = Ground damage divisor
```

Reply:
It was like that so I left it, but if you say that we can remove the quotes then sure let's do it.